arch-test/test.arch-spec.ts

```diff
@@ -0,0 +1,75 @@
+import 'tsarch/dist/jest';
+
+import { filesOfProject } from 'tsarch';
+
+const skipTestFilesPattern = '^(?!.*\\.(spec|int-spec)\\.ts$).*$';
+
+describe('architecture', () => {
+  jest.setTimeout(60000);
```

Comment:
Just curious: what is the timeout for? and why this number precisely? :)

Reply:
It's limit time for this test, it think it's better to add contants to make it more readable